fjord/feedback/static/css/generic_feedback.less

```diff
@@ -135,19 +135,19 @@ body.html-rtl header .header-content .left-side {
 
 #back-button-container {
   display: none;
-  height: 30px;
-  margin: 0;
+  height: 38px;
+  margin: 0px 0px 0px 0px;
   padding: 0;
-  width: 30px;
+  width: 50px;
```

Comment:
You don't want to change the height and width here--that just distorts the image.